testing/vcs/test_boxfill_number_color_more_than_number_levels.py

```diff
@@ -34,6 +34,7 @@
     failed = True
 except:
     failed = False
+    raise
 
```

Comment:
Actually, we do. In case the warning exception isn't thrown, but no others are, we'll need that to catch it.